packages/store/package.json

```diff
@@ -34,40 +34,31 @@
     "test:solidity": "forge test",
     "test:typescript": "tsc --noEmit"
   },
-  "devDependencies": {
+  "dependencies": {
     "@ethersproject/abi": "^5.7.0",
     "@ethersproject/providers": "^5.7.2",
     "@latticexyz/common": "workspace:*",
     "@latticexyz/config": "workspace:*",
     "@latticexyz/schema-type": "workspace:*",
+    "ds-test": "https://github.com/dapphub/ds-test.git#c9ce3f25bde29fc5eb9901842bf02850dfd2d084",
+    "ethers": "^5.7.2",
+    "forge-std": "https://github.com/foundry-rs/forge-std.git#b4f121555729b3afb3c5ffccb62ff4b6e2818fd3"
```

Comment:
I think ds-test and forge can be devDeps as they're just for tests? (console is also imported in normal files, but that's just for debugging and should be eventually removed)